src/style.css

```diff
@@ -0,0 +1,69 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.input {
+  display: flex;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
+  outline: none;
+  font-weight: bold;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.input:focus::placeholder {
+  color: transparent;
+}
+
+.search-form {
+  margin: 20px 0;
+}
+
+.searchbar-big {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+  background-position: 26px 25px;
+  line-height: 22px;
+}
+
+.searchbar-small {
```

Comment:
check this checklist point
<img width="945" alt="image" src="https://user-images.githubusercontent.com/102411130/230295074-c4c00795-96de-4129-90cb-6586f94dc6e1.png">